vignettes/imputation.Rmd

```diff
@@ -43,8 +43,9 @@ The simplest imputation rule is to set the missing parts to a fixed value. For
 example
```

Comment:
In this imputation vignette, i would add a general explanation to the introduction section as follows:
In `{admiral}` we use only two functions `derive_vars_dt()` and `derive_vars_dtm()` for date and datetime imputations respectively. In all other functions where dates can be passed as an argument, we expect full dates or datetimes (unless otherwise specified), so if any possibility of partials then these functions should be used as a first step to make the required imputation.

Reply:
Added